conductr_cli/sandbox_run.py

```diff
@@ -42,15 +42,14 @@ def run(args):
     feature_results = []
     print_proxy_output = False
     if is_conductr_started:
-        if not is_conductr_v1:
-            print_proxy_output = sandbox_proxy.start_proxy(proxy_bind_addr=run_result.core_addrs[0],
-                                                           bundle_http_port=args.bundle_http_port,
-                                                           proxy_ports=sorted(args.ports))
-
         for feature in features:
+            feature.configure(args, run_result)
             result = feature.start()
             feature_results += result
 
+            if isinstance(feature, sandbox_features.ProxyingFeature):
```

Comment:
Yeah, if feels wrong to do a special test. I thought that features had some facility to contribute to the output. Perhaps look at the monitoring feature for inspiration.